care/facility/api/viewsets/prescription.py

```diff
@@ -176,8 +176,7 @@ def list(self, request):
         from care.facility.static_data.medibase import MedibaseMedicineTable
 
         queryset = MedibaseMedicineTable
-
-        limit = request.query_params.get("limit", 30)
+        limit = min(int(request.query_params.get("limit", 30)), 100)
```

Comment:
Won't it raise a `ValueError` if the "limit" query parameter is not a valid integer?

Reply:
yes it will

Reply:
isn't that expected?

Reply:
I mean, it will raise an Exception at the backend rather than catching an invalid Integer and showing a message to the user, right?

Reply:
Got it!